PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.kt

```diff
@@ -364,11 +369,15 @@ class PayPalClient internal constructor(
             return
         }
 
+        braintreeClient.sendAnalyticsEvent(PayPalAnalytics.EDIT_FI_STARTED)
+
         internalPayPalClient.sendVaultEditRequest(context, payPalVaultEditRequest) { result, error ->
             if (error != null) {
+                braintreeClient.sendAnalyticsEvent(PayPalAnalytics.EDIT_FI_FAILED)
                 callback.onPayPalVaultEditAuthRequest(PayPalVaultEditAuthRequest.Failure(error))
             }
 
+            braintreeClient.sendAnalyticsEvent(PayPalAnalytics.EDIT_FI_SUCCEEDED)
```

Comment:
I think instead of firing these at each callback callsite we want to instead use a helper method like we do for the tokenize flow. Let's match a similar pattern to `callbackCreatePaymentAuthFailure`/`callbackBrowserSwitchCancel`/`callbackTokenizeFailure`/`callbackTokenizeSuccess` for edit FI.